pkg/images/defaults/flavor.go

```diff
@@ -190,6 +195,38 @@ func RHACSReleaseImageFlavor() ImageFlavor {
 	}
 }
 
+// OpenSourceReleaseImageFlavor returns image values for `opensource` flavor.
+func OpenSourceReleaseImageFlavor() ImageFlavor {
+	v := version.GetAllVersionsDevelopment()
```

Comment:
If you mean that `opensource` will be a flavor for **release** images, this should rather be `version.GetAllVersionsUnified()`.
If you mean that `opensource` is for development images, could you please share your view on **what should be the image flavor for open source _release_ images**?
Did you have a chance to read https://docs.google.com/document/d/1Gn-tbsyK5133HgIPraZXaOHUAdZ2VnrDnHVwkt1xgY4/edit#heading=h.r8drfclkg6a9 ?

Reply:
Good catch! I intended it to be a flavor for *release* images.
I've checked the document and intended to implement option 4 (opensource as release flavor)